packages/fluentui/react-northstar/package.json

```diff
@@ -38,6 +38,7 @@
     "@types/simulant": "^0.2.0",
     "@uifabric/build": "^7.0.0",
     "faker": "^4.1.0",
+    "jest-watch-typeahead": "^0.4.2",
```

Comment:
@kenotron  or @ecraig12345 should this just go in the scripts/package.json file?

Reply:
I think so